blog/Introducing-Femto.md

```diff
@@ -0,0 +1,184 @@
+
+For example when you install `Fable.React` in your Fable project you also have to install `react` and `react-dom` into your npm dependencies. Users usually have to look up the the exact versions of the npm package they need from the documentation page of the Fable binding itself, in case of `Fable.React` you need to install `react@16.8.0` and `react-dom@16.8.0`.
+
+This is of course doable if you only have a couple of packages with direct npm dependencies such as `Fable.React` but the problem gets even more complicated when you install a nuget package that itself *depends* on a Fable binding such as `Fable.Elmish.React` which depends both on `Fable.Elmish` and `Fable.React`:
+```
+Fable.Elmish.React (nuget)
+  |
+  | -- Fable.Elmish (nuget)
+  | -- Fable.React (nuget)
+        |
+        | -- react@16.8.0 (npm)
+        | -- react-dom@16.8.0 (npm)
+```
+Here, when you install `Fable.Elmish.React`, you also install `Fable.React` because it is a *transitive* dependency. This means now you have to install `react` and `react-dom` yourself because you installed `Fable.React`.
+
+This is something you just have to know and applies to every Fable binding you use. Wouldn't it be nice if you didn't have to do this manually, having a tool that does this for you correctly and automatically? Here comes Femto into play.
+
+### What is Femto
+
+Femto is a dotnet CLI tool that primarily does two things
+ - **Project dependency analysis**: going through all your project's dependencies, the direct and the transitive, extracting information about which npm dependencies are needed, which versions they should have and where they should be installed. Next this information is compared against what you already have installed to decide whether a package is missing or if a version of an installed npm package has an invalid version (falls outside the required version constraint specified by the Fable binding). Femto will log the commands that are required for full package resolution.
+
+ - **Automatic package resolution**: Logging the actions required for package resolution is nice, you can manually execute them one by one and  have all the packages you need in the right place. However why do it manually when Femto can do it for you? Run Femto with the `--resolve` flag and let it do the magic.
+
+Today, Femto is released as beta and we would love to hear from you what you think about it, ready to take it for a spin?
+
+### Installing and using Femto
+
+Install [Femto](https://github.com/Zaid-Ajaj/Femto) as a global dotnet tool as follows:
+```
+dotnet tool install femto --global
+```
+then you can `cd` your way to where you have your Fable project and run project analysis:
+```bash
+# navigate to your project directory
+cd fable-minimal
+cd src
+
+# run femto for project analysis
+femto
+```
+You can also specify the directory of your Fable project or specify the project path itself, the following will find the same the same project.
+```bash
+femto .
```

Comment:
Can't we replace this example with `femto` directly instead of `femto .`?

Reply:
I showed `femto` in the example above it so this is for specifying a directory